tooling/template-base/components/LearnerAccess/LoadArchivedContent.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useCallback } from 'react';
+import { NetworkStatus } from '@apollo/client';
+import { useUserArchivesQuery, LoadingDots, formatTime } from '@thoughtindustries/content';
+import { ReinstateButton } from './MutationCallingButtons';
+import useLearnerAccess from './Context/use-context';
+import { t } from 'i18next';
+const LoadArchivedContent = (): JSX.Element => {
+  const {
+    data,
+    loading,
+    error,
+    refetch: refetchArchives,
+    networkStatus
+  } = useUserArchivesQuery({
+    variables: {},
+    fetchPolicy: 'network-only',
+    notifyOnNetworkStatusChange: true,
+    ssr: false
```

Comment:
Know this was from the other PR too, but we can omit the `ssr: false` here I believe